jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/WebAppContext.java

```diff
@@ -1251,6 +1250,7 @@ protected void stopContext() throws Exception
 
             _unavailableException = null;
         }
+        stopWebapp();
```

Comment:
I think something like this is the only change we need to make in this PR.

Reply:
Since `stopWebapp()` must be the last thing done by this method, this should probably be reflected by a comment explaining why.